lib/jmap/mail/email/get/get_email_method.dart

```diff
@@ -0,0 +1,86 @@
+import 'package:jmap_dart_client/http/converter/account_id_converter.dart';
+import 'package:jmap_dart_client/http/converter/id_converter.dart';
+import 'package:jmap_dart_client/http/converter/properties_converter.dart';
+import 'package:jmap_dart_client/http/converter/unsigned_int_nullable_converter.dart';
+import 'package:jmap_dart_client/jmap/account_id.dart';
+import 'package:jmap_dart_client/jmap/core/capability/capability_identifier.dart';
+import 'package:jmap_dart_client/jmap/core/method/request/get_method.dart';
+import 'package:jmap_dart_client/jmap/core/request/request_invocation.dart';
+import 'package:jmap_dart_client/jmap/core/request/result_reference.dart';
+import 'package:jmap_dart_client/jmap/core/unsigned_int.dart';
+import 'package:jmap_dart_client/jmap/mail/email/email_body_properties.dart';
+import 'package:json_annotation/json_annotation.dart';
+
+part 'get_email_method.g.dart';
+
+@UnsignedIntNullableConverter()
+@IdConverter()
+@AccountIdConverter()
+@PropertiesConverter()
+@JsonSerializable()
```

Comment:
please use `@JsonSerializable(explicitToJson: true)`, to support nested class `toJson` also.
```
..referenceIds = json['#ids'] == null
        ? null
        : ResultReference.fromJson(json['#ids'] as Map<String, dynamic>)
```